src/routers/datatypeRouters/nodes/variants.ts

```diff
@@ -201,17 +201,49 @@ async function conditionalSearch (input: paramsFormatType): Promise<any[]> {
   return await (await db.query(query)).all()
 }
 
+export async function variantIDSearch (input: paramsFormatType): Promise<any[]> {
+  let variantSchema = humanVariantSchema
+  if (input.organism === 'Mus musculus') {
+    variantSchema = mouseVariantSchema
+  }
+  delete input.organism
+
+  let useIndex = ''
+  if (input.chr !== undefined && input.position !== undefined) {
+    input.region = `${input.chr}:${input.position}-${input.position}`
+    useIndex = 'OPTIONS { indexHint: "region", forceIndexHint: true }'
+    delete input.chr
+    delete input.position
+  }
+
+  let filterBy = ''
+  const filterSts = getFilterStatements(variantSchema, preProcessVariantParams(input))
+  if (filterSts !== '') {
+    filterBy = `FILTER ${filterSts}`
+  } else {
+    return []
+  }
+  const query = `
+    FOR record IN ${variantSchema.db_collection_name as string} ${useIndex}
+    ${filterBy}
+    SORT record._key
+    LIMIT 0, ${QUERY_LIMIT}
+    RETURN record._id
+  `
+  return await (await db.query(query)).all()
+}
+
```

Comment:
this is the equivalent of the `variantSearch` method, like:
`variant = variantSearch(input)[0].id`

Reply:
try to test it but got an error: `Element implicitly has an 'any' type because expression of type '0' can't be used to index type 'Promise<any[]>'. Property '0' does not exist on type 'Promise<any[]>'.ts(7053)`